docs/etcd_clusters.md

```diff
@@ -0,0 +1,60 @@
+# Manage etcd clusters
+
+Mayu contains an etcd discovery to setup and manage your etcd clusters. During startup you can configure if you like to use an external etcd discovery or Mayu itself.
```

Comment:
We can't replace `--etcd-endpoint=http://localhost:2379` because that is not the discovery but the etcd connection that is necessary for Mayus discovery. 

You are talking about `--etcd-discovery=http://localhost:4080` which is the default (but using mayus ip). That's the reason why I left it out here.